birdie_snapshots/inline_object_schema_test.accepted

```diff
@@ -20,17 +20,17 @@ pub fn user_decoder() {
     None,
     decode.optional(decode.bool),
   )
-  use name <- decode.optional_field("name", None, decode.optional(decode.string))
   use age <- decode.optional_field("age", None, decode.optional(decode.int))
-  decode.success(User(active: active, name: name, age: age))
+  use name <- decode.optional_field("name", None, decode.optional(decode.string))
+  decode.success(User(active: active, age: age, name: name))
 }
 
 pub fn user_encode(data: User) {
   json.object(
     [
       #("active", json.nullable(data.active, json.bool)),
-      #("name", json.nullable(data.name, json.string)),
-      #("age", json.nullable(data.age, json.int))
+      #("age", json.nullable(data.age, json.int)),
+      #("name", json.nullable(data.name, json.string))
     ],
```

Comment:
Don't know why this changed, could be the versions of the standard lib. Not sure. 
But it is just the order.

Reply:
I'm curious why this changed but maybe I just forgot to commit a file earlier